### DISCUSSION TITLE
TTS

### DISCUSSION THREAD

**Discussion by forwaid** [ReadYouApp/ReadYou#188] [Q&A]
Does it support calling Android system TTS to read aloud? 

The application is very good, the layout is good, the UI is simple and beautiful, thank you for your efforts to let us experience such a good Rss reader.

**Comment by bigdiff** [comment]
Currently it does not support performing text to speech.

There is an open issue if you wish to subscribe: https://github.com/Ashinch/ReadYou/issues/42
